src/style.css

```diff
@@ -1 +1,94 @@
 /* Styles go here */
+:root {
+  --header-height: 60px;
+  --line-height: 14.6px;
+}
+
+.header {
+  width: 1024px;
+  margin: auto 0;
+  height: var(--header-height);
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  position: relative;
+  justify-content: space-around;
+}
+
+.nav {
+  position: absolute;
+  right: 50px;
+  height: 100%;
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+}
+
+.moyo {
+  height: 40px;
+  width: 40px;
+  position: absolute;
+  left: 50px;
+}
+
+.nav__list {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: var(--header-height);
+  text-transform: uppercase;
+  color: #000;
+  padding: 0;
+  list-style-type: none;
+  display: flex;
+  flex-direction: row;
+  margin: 0;
+  align-items: center;
+  text-align: center;
+}
+
+.nav__item {
+  margin-left: 20px;
+  position: relative;
+  height: 100%;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  display: block;
+  text-decoration: none;
+  color: black;
+  height: 100%;
+}
+
+.nav__link:hover {
+  cursor: pointer;
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  content: "";
+  position: absolute;
+  width: 100%;
+  transform: scaleX(1);
+  border-bottom: 4px solid #00acdc;
+  border-radius: 8px;
+  bottom: 0;
+  left: 0;
+}
+
+@media (min-width: 1200px) {
+  .header {
+    width: 1200px;
+  }
+}
```

Comment:
No need to use media in this task. It's okay if header will cover 100% width on big screens